lib/seattleflu/api/routes.py

```diff
@@ -3,17 +3,21 @@
 """
 import logging
 from flask import Blueprint, request, send_file
+from flask_deprecate import deprecate_blueprint
```

Comment:
I'm not sure flask-deprecate really has much use for us here. The `Warnings` header it adds (which is the only thing it does) is likely to be ignored by most clients. In my experience, it's not a header which sees wide use and is typically only used in caches. Thoughts?

Reply:
I wasn't sure how else to warn the user of an endpoint deprecation. Do you have other ideas? I also looked at `flask-socketio`.

Reply:
Ah, I think we warn them by talking to them in person or via Slack. :-) We can deploy this and get people to shift endpoints later, which might wait until we have a /v2 we want them to use so they go / → /v2 instead of / → /v1 (but same behaviour) → /v2.

(socketio looks unrelated to me; it's about websockets and related protocols.)